patterns/base/pre/sass/partials/_extends.scss

```diff
@@ -0,0 +1,27 @@
+%base--pre-build {
+  @extend %font--mono;
+  display: block;
+  hyphens: none;
+  word-spacing: normal;
+  text-align: left;
+  word-break: normal;
+  white-space: pre;
+  direction: ltr;
+  tab-size: 2;
+}
+
+%base--pre {
+  @extend %base--pre-build;
+  margin: .5em 0;
+  overflow: auto;
+
+  border: 1px solid rgba(color('gray'), .25);
+
+  code,
+  .base--code {
+    @extend %base--pre-build;
+    padding: 0;
+    font-size: 1em;
```

Comment:
Wadaya mean set explicitly?

We need to update how code works when nested inside `pre`, which is different than just inline `code`